packages/teko/tests/unit_tests/tProbingFactory.cpp

```diff
@@ -68,7 +68,9 @@
 #include "Teko_DiagonallyScaledPreconditionerFactory.hpp"
 #include "Teko_PreconditionerInverseFactory.hpp"
 #include "Teko_PreconditionerLinearOp.hpp"
+#ifdef TEKO_HAVE_EPETRA
 #include "Teko_ProbingPreconditionerFactory.hpp"
+#endif
```

Comment:
Changes to this file are probably not needed. This entire file/test is removed by cmake if epetra is disabled in teko.

Reply:
Good call, I'll remove this part of the change. The other file was the issue, I was just grepping for other instances of the same pattern and didn't catch the build system guard.